molecule_ec2/cookiecutter/{{cookiecutter.molecule_directory}}/{{cookiecutter.scenario_name}}/create.yml

```diff
@@ -73,53 +73,49 @@
         filters:
           name: "{{ item.image_name }}"
       loop: "{{ molecule_yml.platforms }}"
-      when: item.image is not defined
+      when: item.image_id is not defined
```

Comment:
I'm not sure switching from `item.image` to `item.image_id` is worthwhile

If it is, it should probably be a separate pull request

It doesn't align with the documentation, it also hurts potential upgrading by requiring the molecule.yml file to be updated